app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -238,7 +238,7 @@ class StateFragmentTest {
     TestPlatformParameterModule.forceEnableContinueButtonAnimation(false)
     Intents.init()
     setUpTestApplicationComponent()
-    testCoroutineDispatchers.registerIdlingResource()
+//    testCoroutineDispatchers.registerIdlingResource()
```

Comment:
Did you mean to commit this? We shouldn't check in commented out code, so please revert.

Reply:
I totally forgot about this comment, apologies for this.

Reply:
No worries.